game-core/src/main/java/games/strategy/triplea/ui/mapdata/MapData.java

```diff
@@ -500,8 +500,10 @@ public Color getPlayerColor(final String playerName) {
 
   /** returns the color for impassable territories. */
   public Color impassableColor() {
-    // just use getPlayerColor, since it parses the properties
-    return getPlayerColor(Constants.PLAYER_NAME_IMPASSABLE);
+    return Optional.ofNullable(
+            getColorProperty(PROPERTY_COLOR_PREFIX + Constants.PLAYER_NAME_IMPASSABLE))
+        .or(() -> Optional.ofNullable(getColorProperty(PROPERTY_COLOR_PREFIX + "Impassible")))
+        .orElseGet(defaultColors::nextColor);
```

Comment:
Off-Topic: I don't think we have a nice error message for "too few default colors" yet, so that might be something to consider here

Reply:
Nevermind, just saw https://github.com/triplea-game/triplea/pull/7637